lib/scarlet/helpers/nick_helper.rb

```diff
@@ -0,0 +1,57 @@
+class Scarlet
+  module NickHelper
+    def handle_special_nick(name)
+      case name
+      when ':me'
+        sender.nick
+      when ':you'
+        server.current_nick
+      else
+        name
+      end
+    end
+
+    def find_nicks(names)
+      names.words.map do |name|
+        name = handle_special_nick(name)
+        Scarlet::Nick.first(nick: name)
+      end
+    end
+
+    def find_nick(name)
+      find_nicks(name).first
```

Comment:
Nah, `find_nicks` is intended to produce a list of nicks, which is why we have `find_nick` for one, the conditions for getting a list of nicks is "nick1 nick2", and commands that expect a list will most likely use `/command name (?<nicks>.+)/` instead of `/command name (?<nick>\S+)/`